docs/setup/configuration.mdx

```diff
@@ -270,6 +270,18 @@ Controls [Django's secret key](https://docs.djangoproject.com/en/3.0/ref/setting
 
 Controls the production assets mount path. Defaults to `/static/`.
 
+### `BREAKER_BOARD_ENABLED`
+
+Enables the [circuit breaker](developer/extending/webhooks/synchronous-events/circuit-breakers.mdx) feature for sync webhooks.
+
+### `BREAKER_BOARD_SYNC_EVENTS`
+
+List of comma separated, lowercase sync webhook events, which should be monitored by the breaker board.
```

Comment:
We could provide some examples or, if possible, link to the list values that could be set here.